pkg/virt-config/featuregate/feature-gates.go

```diff
@@ -28,11 +28,16 @@ import (
 type State string
 
 const (
-	// By default, GAed feature gates are considered enabled and no-op.
-	GA = "General Availability"
-	// The feature is going to be discontinued next release
-	Deprecated     = "Deprecated"
-	Discontinued   = "Discontinued"
+	// GA represents features that reached General Availability.
+	// GA features are considered enabled, with no option to disable them.
```

Comment:
This is simply not true imho

Reply:
You need to elaborate here.
A GA feature is always on, you are unable to set it off through a FG once it is GA.

Reply:
Well a feature can have configuration knob, right? The text you provided leads users to believe there is no other option...

Reply:
We are under the `featuregate` package, that should provide the context of discussing feature gates and not features. But assumption from my side may not be in sync with how things are perceived.

If you still think it is not clear and requires clarifying that it discusses the FG flag, then I will change this.